src/services/openletters/index.jsx

```
@@ -35,43 +33,58 @@ export const AddOpenLetter = ({
   const { bgColor } = useSelector((state) => state.theme);
 
   const imageRef = useRef(null);
+  const sigCanvas = useRef(null);
+  const videoRef = useRef(null);
 
   const [subCategory, setSubCategory] = useState([]);
   const [isLoading, setIsLoading] = useState(false);
```

ESLint

[error] 'setIsLoading' is assigned a value but never used.

(no-unused-vars)